index.css

```diff
@@ -735,6 +768,74 @@ footer {
   background-color: hsla(22, 6%, 10%, 1);
 }
 
+.project-text-div {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  left: 5%;
+  background-color: #fff;
+  width: 84vw;
+  padding: 16px 16px 24px 16px;
+}
+
+.project-text-div img {
+  width: 50%;
+  height: auto;
+}
+
+.myBck {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-around;
+  width: 153px;
+}
+
+.title-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#popup-close-btn {
+  font-size: 40px;
+}
+
+.popup-description-container {
+  display: flex;
+  flex-direction: column-reverse;
+  align-self: flex-start;
+  justify-content: flex-end;
+}
+
+.popup-description-container p {
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  color: hsla(218, 76%, 15%, 1);
+  margin-bottom: 20px;
+  width: 300px;
+}
+
+.project-image-div {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.project-image-div img {
+  width: 100%;
+  height: auto;
+}
+
+#template {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 3589px;
+  background-color: rgba(0, 0, 0, 0.7);
+  display: none;
+}
```

Comment:
- [ ] Great job so far 🍾 , the popup looks great on the desktop screen 👏.However, there are some issues when you open the popup on the mobile screens. It is not aligned well and doesn't fit the screen well as it is shown on the Figma design. Please take a look at the comparison by the screenshot below and fix it..
**Your Design** 
![image](https://user-images.githubusercontent.com/90222110/180083248-689af734-c91f-4c1a-bb75-905f5498557c.png)
**Figma design**
![image](https://user-images.githubusercontent.com/90222110/180083467-d5ed712b-5b55-4234-99dc-025e3757cd85.png)

- [ ] Note that the background of the popup is both dark and blurred 😺. Please implement them too in your design

**NOTE:** When the popup is present on the screen, the user should not be able to scroll. This helps them to keep focusing on the popup so please make sure you hide the scroll bars too 😃 👍

Reply:
###Hello, Thank you for reviewing my project.
- I've Implemented the requested changes but however, but I have a problem.
Can you please elaborate on what you mean to align it perfectly for the mobile? 
For me, it is aligned Perfectly (360px of width and above). I don't  think any mobile phone has a lower width than 360 px
![proof](https://user-images.githubusercontent.com/104627775/180181057-9db792c3-78ab-446a-bf1a-0bac146a21a8.PNG)

Reply:
### Hello, I am still waiting for your response on how to align the image? In my case, it is still aligned properly at 360 px of screen width.
![notMyRealFault](https://user-images.githubusercontent.com/104627775/180238009-17827831-9701-4bf7-9177-c86d9b7f0d06.PNG)